velocity/src/main/java/me/leoko/advancedban/velocity/VelocityMain.java

```diff
@@ -55,6 +56,10 @@ public void onProxyInitialization(ProxyInitializeEvent event) {
         server.getEventManager().register(this, new ConnectionListenerVelocity(server));
         server.getEventManager().register(this, new ChatListenerVelocity());
 
+        //If signed velocity is missing send a warning
+        if (!server.getPluginManager().getPlugin("signedvelocity").isPresent()) {
+            logger.warn("SignedVelocity is not installed, mute system may kick your players while muted. Please install SignedVelocity to prevent this.");
```

Comment:
Further limit the use of the mute feature with this. Refer to my other comment:

> Is it maybe possible to implement in the following way:

> Check if SignedVelocity is installed
If SignedVelocity: Check if message signing is enabled (secure-profiles iirc)
if both fail, entirely block the use of the mute feature, such as by erroring out on the mute command.
This is to ensure that confusion would not arise when setting up a velocity server.

> This could for example be done by implementing a "enableMutes" which is only enforced by velocity, and is simply set to true on everything else by default except Velocity.